src/rpdk/core/__init__.py

```diff
@@ -1,5 +1,5 @@
 import logging
 
-__version__ = "0.1.14"
+__version__ = "0.2.0"
```

Comment:
https://github.com/aws-cloudformation/cloudformation-cli-java-plugin/blob/master/setup.py#L39 important to note this. might need to bump this